### ISSUE TITLE
[Feature Request]: Align and document identity feature across modules

### ISSUE THREAD

**Issue by eriqua** [Azure/ResourceModules#2992]
### Description

The identity feature should be aligned across all modules supporting it (parameters, variables, resource setting and outputs naming and implementation).

A dedicated paragraph should be added to the module design patterns section.

**Comment by rahalan** [comment]
Compare implementations and decide for best one, before implementing for all modules

**Comment by AlexanderSehr** [comment]
Should align to AVM specs

**Comment by AlexanderSehr** [comment]
Hey @eriqua, I wonder what to do with this. I guess each spec in AVM should justify why something is done a certain way (which I doubt it is). Do you think it would make sense to transition it to the AVM repository?

**Comment by eriqua** [comment]
@AlexanderSehr I agree, it's probably one to move to AVM repo, documentation related, rather than BRM, since the implementation 'should' already be aligned

**Comment by AlexanderSehr** [comment]
Alright, will transfer it then. Let's see if anybody picks it up 😉
